packages/sdk-ui-ts/validators-logo/index.ts

```diff
@@ -0,0 +1,256 @@
+import { Validator, ChainGrpcStakingApi } from '@injectivelabs/sdk-ts'
+import { Network, getNetworkEndpoints } from '@injectivelabs/networks'
+import { HttpRestClient } from '@injectivelabs/utils'
+import path from 'path'
+import fs from 'fs'
+// @ts-ignore
+import validatorToAddressMapFromKeybase from './mappings.json'
+import { validatorAddressToPathMap } from '../src/utils/mappings'
+
+export interface ValidatorMap {
+  [validatorAddress: string]: string
+}
+
+const endpoints = getNetworkEndpoints(Network.MainnetK8s)
+const chainGrpcStakingApi = new ChainGrpcStakingApi(endpoints.grpc)
+const keybaseApi = new HttpRestClient('https://keybase.io/_/api/1.0/')
+
+const KEYBASE_SUFFIX = 'user/lookup.json?fields=pictures&key_suffix='
+
+const existingValidatorToAddressMapFromKeybase =
+  { ...JSON.parse(JSON.stringify(validatorToAddressMapFromKeybase)) } || {}
+
+/* cli flags */
+const [, , flag] = process.argv
+const cliValidatorAddress = flag && flag.includes('inj') ? flag : undefined
+const shouldUpdateAllImages = flag && flag.includes('update:all')
```

Comment:
Lets rename the flags to follow node.the js cli arguments convention
https://stackabuse.com/command-line-arguments-in-node-js/
```
yarn validator-logos --update-all 
yarn validator-logos --validator-address="inj...."
```